lib/generated/assets.dart

```diff
@@ -0,0 +1,29 @@
+///This file is automatically generated. DO NOT EDIT, all your changes would be lost.
+class Assets {
```

Comment:
Remove this file and use the assets file we have, no need to deal with the auto generated files